lib/main.dart

```diff
@@ -332,7 +332,10 @@ class CampusAppState extends State<CampusApp> with WidgetsBindingObserver {
       builder: Provider.of<SettingsHandler>(context).currentSettings.useSystemTextScaling
           ? null
           : (context, child) {
-              return MediaQuery(data: MediaQuery.of(context).copyWith(textScaleFactor: 1), child: child!);
+              return MediaQuery(
+                data: MediaQuery.of(context).copyWith(textScaler: TextScaler.noScaling),
```

Comment:
Good point. This was already adressed within https://github.com/astarub/campus_app/pull/160 but I forgot to check the migration guide.

Note: https://docs.flutter.dev/release/breaking-changes/deprecate-textscalefactor#examples